#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_fix_privacy_breach_logo.dpatch by Pau Garcia i Quiles <pgquiles@elpauer.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not download SourceForge and GitHub logos from the Internet

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' witty-3.3.11+dfsg~/examples/wt-homepage/jwt-home.xml witty-3.3.11+dfsg/examples/wt-homepage/jwt-home.xml
--- witty-3.3.11+dfsg~/examples/wt-homepage/jwt-home.xml	2018-10-22 15:06:57.000000000 -0400
+++ witty-3.3.11+dfsg/examples/wt-homepage/jwt-home.xml	2018-10-22 15:10:18.543824151 -0400
@@ -587,7 +587,6 @@
 
 <h4>Git repository</h4>
 
-<a href="https://github.com/kdeforche/jwt"><img style="display: block; width: 80px; float: right" src="https://github.com/github/media/blob/master/octocats/octocat.png?raw=true"></img></a>
 <p>If you want to keep track of the latest changes, or participate in
 JWt development, you may want to work from
 the <a href="https://github.com/kdeforche/jwt">github
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' witty-3.3.11+dfsg~/examples/wt-homepage/wt-home.xml witty-3.3.11+dfsg/examples/wt-homepage/wt-home.xml
--- witty-3.3.11+dfsg~/examples/wt-homepage/wt-home.xml	2018-10-22 15:06:57.000000000 -0400
+++ witty-3.3.11+dfsg/examples/wt-homepage/wt-home.xml	2018-10-22 15:10:18.543824151 -0400
@@ -1066,7 +1066,6 @@
 
 <h4>Git repository</h4>
 
-<a href="https://github.com/kdeforche/wt"><img style="display: block; width: 80px; float: right" src="https://github.com/github/media/blob/master/octocats/octocat.png?raw=true"></img></a>
 <p>If you want to keep track of the latest changes, or participate in
 Wt development, you may want to work from
 the <a href="https://github.com/kdeforche/wt">github
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' witty-3.3.11+dfsg~/examples/wt-homepage/wt-home.xml.rej witty-3.3.11+dfsg/examples/wt-homepage/wt-home.xml.rej
--- witty-3.3.11+dfsg~/examples/wt-homepage/wt-home.xml.rej	1969-12-31 19:00:00.000000000 -0500
+++ witty-3.3.11+dfsg/examples/wt-homepage/wt-home.xml.rej	2018-10-22 15:10:18.543824151 -0400
@@ -0,0 +1,10 @@
+--- examples/wt-homepage/wt-home.xml	2016-07-13 14:27:26.000000000 +0200
++++ examples/wt-homepage/wt-home.xml	2016-09-25 22:39:52.978699430 +0200
+@@ -1197,7 +1196,6 @@ The Chinese translation of the homepage
+ 
+ <h4>Sourceforge</h4>
+ 
+-<a href="http://sourceforge.net"><img src="https://sflogo.sourceforge.net/sflogo.php?group_id=153710&amp;type=1" style="vertical-align:middle" width="88" height="31" border="0" alt="SourceForge.net Logo" /></a>
+ The Wt project is hosted at sourceforge <a href="http://sourceforge.net/projects/witty/">here</a>.
+   </message>
+   <message id="home.other-language">
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' witty-3.3.11+dfsg~/examples/wt-homepage/wt-home_cn.xml witty-3.3.11+dfsg/examples/wt-homepage/wt-home_cn.xml
--- witty-3.3.11+dfsg~/examples/wt-homepage/wt-home_cn.xml	2018-10-22 15:06:57.000000000 -0400
+++ witty-3.3.11+dfsg/examples/wt-homepage/wt-home_cn.xml	2018-10-22 15:10:18.543824151 -0400
@@ -913,7 +913,6 @@
 
 <h4>Sourceforge</h4>
 
-<a href="http://sourceforge.net"><img src="https://sflogo.sourceforge.net/sflogo.php?group_id=153710&amp;type=1" style="vertical-align:middle" width="88" height="31" border="0" alt="SourceForge.net Logo" /></a>
 Wt在Sourceforge的主页请点击<a href="http://sourceforge.net/projects/witty/">此处</a>。
   </message>
   <message id="home.other-language">
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' witty-3.3.11+dfsg~/examples/wt-homepage/wt-home_ru.xml witty-3.3.11+dfsg/examples/wt-homepage/wt-home_ru.xml
--- witty-3.3.11+dfsg~/examples/wt-homepage/wt-home_ru.xml	2018-10-22 15:06:57.000000000 -0400
+++ witty-3.3.11+dfsg/examples/wt-homepage/wt-home_ru.xml	2018-10-22 15:10:18.543824151 -0400
@@ -1212,7 +1212,6 @@
 
 <h4>Sourceforge</h4>
 
-<a href="http://sourceforge.net"><img src="https://sflogo.sourceforge.net/sflogo.php?group_id=153710&amp;type=1" style="vertical-align:middle" width="88" height="31" border="0" alt="SourceForge.net Logo" /></a>
 Проект Wt размещён на sourceforge <a href="http://sourceforge.net/projects/witty/">здесь</a>.
   </message>
   <message id="home.other-language">
